---
permalink: /
title: "About me"
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

I am a tenure-track assistant professor (AI Initiative) at the Department of Health Outcome and Biomedical Informatics, University of Florida. I completed my postdoctoral training at the Department of Biomedical Informatics, Harvard Medical School, working with <a href="https://avillach-lab.hms.harvard.edu/">Dr. Paul Avillach</a>. Before that, I received my Ph.D. degree from School of Computer Science and Engineering, Nanyang Technological University in 2020, under the supervison of <a href="https://personal.ntu.edu.sg/asckkwoh/">Dr. Kwoh Chee Keong</a>. My thesis is meta-analysis on the lethality of influenza A viruses using machine learning approaches, examined by dissertation advisory committee members <a href="https://personal.ntu.edu.sg/boan/">Assoc. Professor Bo An</a>,  <a href="https://sentic.net/erikcambria/">Assoc. Professor Erik Cambria</a> and <a href="https://personal.ntu.edu.sg/sinnopan/">Assoc. Professor Sinno Jialin Pan</a>. I obtained my bachelor's degree and master's degree from Shandong University and Central South University in 2015 and 2013, respectively.

<b>Research Interest</b>
- Computational modeling in RNA viruses (influenza, coronaviruses, zika, etc.)  
- Machine learning modeling to facilitate the diagnosis of rare/undiagnosed diseases
- Genotype-phenotype association study for biomarker identification and novel discovery
- Combination of clinical informatics and genomics to support clinical decision system
- Interpretable and trustworthy AI to advance health equity
 
<font size="5"><b>Recent News</b></font>
- [Nov.2022] Our paper "GraphLncLoc: long non-coding RNA subcellular localization prediction using graph convolutional networks based on sequence to graph transformation" was accepted by Briefings in Bioinformatics.  
- [Nov.2022] I am serving as Program Committee member for the 21th Asia Pacific Bioinformatics Conference (APBC).
- [Nov.2022] Xiangyu Chen joined our lab as a research intern, welcome!
- [Oct.2022] Jiaxin Li joined our lab as a research undergraduate student, welcome!
- [Oct.2022] Our collaborative proposal "Computational Drug Repurposing for AD/ADRD with Integrative Analysis of Real-World Data and Biomedical Knowledge was funded by NIA!
- [Sep.2022] I am serving as Program Committee member for IEEE International Conference on Bioinformatics and Biomedicine (IEEE BIBM 2022).
- [Aug.2022] Xiaohan Li joined our lab as research graduate student, welcome!
- [Jul.2022] Peiyi Zheng joined our lab as research intern, welcome!
- [Jun.2022] Two Ph.D. students Biao Ye and Changyu Yin joined our lab, welcome!
- [Jun.2022] Our paper â€œA framework for predicting variable-length epitopes of human-adapted viruses using machine learning methods" was accepted by Briefings in Bioinformatics.
- [May.2022] I was appointed as guest editor for Research Topic: Computational Methods to Analyze RNA Data for Human Diseases in the journal of Frontiers In Genetics.
- [May.2022] I accpeted the offer from the University of Florida and will join the university as a tenure-track assistant professor this November.
- [Apr.2022] I was invited to give a talk at Auburn University, Auburn, Alabama.
- [Apr.2022] I was invited to give a talk at the University of Florida, Gainesville, Florida.
- [Apr.2022] I was invited to give a talk at Emory University, Atlanta, Georgia.
- [Mar.2022] New preprint "ViPal: A Framework for Virulence Prediction of Influenza Viruses with Prior Viral Knowledge Using Genomic Sequences" is released.
- [Mar.2022] I was invited to give a talk at the University of California Irvine. (Virtual)
- [Mar.2022] I was invited to give a talk at the University of Massachusetts, Boston, Massachusetts.
- [Mar.2022] Our paper "Mendelian Randomization Analysis Suggests No Associations of Herpes Simplex Virus Infections With Multiple Sclerosis" was publised at Frontiers in Neuroscience.
- [Feb.2022] I was invited to give a talk at Boston University, Boston, Massachusetts.    <br />
<br />
<br />

<script type="text/javascript" id="clustrmaps" src="//clustrmaps.com/map_v2.js?d=n9KHZlK5EQXBBjgFgYGJoe_HJrPsBYTt9mMNoLZNZ5c&cl=ffffff&w=500"> 
</script>
