---
#layout: archive
#title: "Publications"
permalink: /publications/
author_profile: true
---

You can also find my articles on <a href="https://scholar.google.com/citations?user=MZigJDUAAAAJ&hl=en">my Google Scholar profile</a>

<font size="5"><b>Journals</b></font>  
- <strong>Yin R</strong>, Luo Z, Kwoh C K. Alignment-free machine learning approaches for the lethality prediction of potential novel human-adapted coronavirus using genomic nucleotide[J]. bioRxiv, 2020. <a href="https://www.biorxiv.org/content/10.1101/2020.07.15.176933v1.full.pdf">(pdf)</a>
- <strong>Yin R</strong>, Luo Z, Zhuang P, et al. VirPreNet: a weighted ensemble convolutional neural network for the virulence prediction of influenza a virus using all 8 segments[J]. Bioinformatics. <a href="https://drive.google.com/file/d/1aykYKyApKUB2nSQ-5F-n16EEUUluvewA/view?usp=sharing">(pdf)</a>
- <strong>Yin R</strong>, Thwin N N, Zhuang P, et al. IAV-CNN: a 2D convolutional neural network model to predict antigenic variants of influenza A virus[J]. bioRxiv, 2020. <a href="https://www.biorxiv.org/content/10.1101/2020.07.15.204883v1.full.pdf">(pdf)</a>
- Lin Z , Feng L, Guo X, <strong>Yin R</strong>, Kwoh CK, Xu X. COMET: Convolutional Dimension Interaction for Deep Matrix Factorization[J]. bioRxiv, 2020. <a href="https://arxiv.org/pdf/2007.14129.pdf">(pdf)</a>
- <strong>Yin R</strong>, Luusua E, Dabrowski J, et al. Tempel: Time-series Mutation Prediction of Influenza A Viruses via Attention-based Recurrent Neural Networks[J]. Bioinformatics, 2020. <a href="https://drive.google.com/file/d/1Gh179fn77z95QaKZxAB9EegKYgBCEMOI/view?usp=sharing">(pdf)</a>
- Zhang Y, Long Y, <strong>Yin R</strong>, et al. DL-CRISPR: A Deep Learning Method for Off-Target Activity Prediction in CRISPR/Cas9 With Data Augmentation[J]. IEEE Access, 2020, 8: 76610-76617. <a href="https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=9076075">(pdf)</a>
- <strong>Yin R</strong>, Zhang Y, Zhou X, et al. Time series computational prediction of vaccines for influenza A H3N2 with recurrent neural networks[J]. Journal of Bioinformatics and Computational Biology, 2020. <a href="https://drive.google.com/file/d/1J-GRWUcammurn0NZWtl5hZ0DFS1jeXC2/view?usp=sharing">(pdf)</a>
- <strong>Yin R</strong>,, Zhou X, Rashid S, et al. HopPER: an adaptive model for probability estimation of influenza reassortment through host prediction[J]. BMC Medical Genomics, 2020, 13(1): 9. <a href="https://bmcmedgenomics.biomedcentral.com/articles/10.1186/s12920-019-0656-7">(pdf)</a>
- Lin Z, Feng L, <strong>Yin R</strong>, et al. GLIMG: Global and Local Item Graphs for Top-N Recommender Systems[J]. bioRxiv, 2020. <a href="https://arxiv.org/pdf/2007.14018.pdf">(pdf)</a>
- Zhou X, <strong>Yin R</strong>, Zheng J, et al. An encoding scheme capturing generic priors and properties of amino acids improves protein classification[J]. IEEE Access, 2019, 7: 7348-7356. <a href="https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=8594660">(pdf)</a>
- <strong>Yin R</strong>, Tran V H, Zhou X, et al. Predicting antigenic variants of H1N1 influenza virus based on epidemics and pandemics using a stacking model[J]. PloS one, 2018, 13(12): e0207777. <a href="https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0207777&type=printable">(pdf)</a>
- Zhou X, <strong>Yin R</strong>, Kwoh C K, et al. A context-free encoding scheme of protein sequences for predicting antigenicity of diverse influenza A viruses[J]. BMC genomics, 2018, 19(10): 936. <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5282-9">(pdf)</a>
- <strong>Yin R</strong>, Zhou X, Zheng J, et al. Computational identification of physicochemical signatures for host tropism of influenza A virus[J]. Journal of bioinformatics and computational biology, 2018: 1840023- 1840023. <a href="https://drive.google.com/file/d/1IWHPJdqRLcJts6hhk5AoHVprtXtvghd-/view?usp=sharing">(pdf)</a>
- Ding P, <strong>Yin R</strong>, Luo J, et al. Ensemble Prediction of Synergistic Drug Combinations Incorporating Biological, Chemical, Pharmacological and Network Knowledge[J]. IEEE journal of biomedical and health informatics, 2018. <a href="https://drive.google.com/file/d/1PRoRS9bG3GCTWEfiwV6cCiu-ue9clStU/view?usp=sharing">(pdf)</a>

<font size="5"><b>Conferences</b></font>  
- <strong>Yin R</strong>, Tan J, Akhila D, et al. Inference of Sequence Homology by BLAST visualization of Influenza Genome set[C]//Proceedings of the 9th International Conference on Computational Systems-Biology and Bioinformatics. ACM, 2018: 5. <a href="https://drive.google.com/file/d/1ZDVAb8Sbh9RJzztc2JLeAQ9V_ZA7ugNP/view?usp=sharing">(pdf)</a>
- Ivan F X, Zhou X, Deshpande A, <strong>Yin R</strong>, et al. Phylogenetic tree based method for uncovering co- mutational site-pairs in influenza viruses[C]//Proceedings of the 8th ACM International Conference on Bioinformatics, Computational Biology, and Health Informatics. 2017: 21-26. <a href="https://drive.google.com/file/d/1QFB7XJnoSvioAQd1zzhKG1oM2NhCmvjl/view?usp=sharing">(pdf)</a>
- <strong>Yin R</strong>, Zhou X, Ivan F X, et al. Identification of Potential Critical Virulent Sites Based on Hemagglu- tinin of Influenza a Virus in Past Pandemic Strains[C]//Proceedings of the 6th International Conference on Bioinformatics and Biomedical Science. ACM, 2017: 30-36. (<strong>Best presentation</strong>) <a href="https://drive.google.com/file/d/1xcg31PhgY7DVH1FsRg3AX9uL2ySfUJ7p/view?usp=sharing">(pdf)</a>

<font size="5"><b>Chapters</b></font> 
- <strong>Yin R</strong>, Chee Keong Kwoh, and Jie Zheng. Whole Genome Sequencing Analysis: Computational Pipelines and Workflows in Bioinformatics, Encyclopedia of Bioinformatics and Computational Biology, 176-183, 2019. <a href="https://drive.google.com/file/d/17rcMUkSofgoFGW3xtz6UOaqoDi3MXU07/view?usp=sharing">(pdf)</a>

<font size="5"><b>Patents</b></font> 
- The device of matrix sedimentation, China, 2014
- The device and method of Linear sweep polarographic wave model, China, 2013
